typegen/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-ndc-client = { git = "http://github.com/hasura/ndc-spec.git", tag = "v0.1.0-rc.15" }
+ndc-client = { git = "http://github.com/hasura/ndc-spec.git", rev = "bfb2fc8c4c21d9f7ee4e4f84daa018ebc57135cb" }
```

Comment:
No tag for this?

Reply:
No, the tag was made before I discovered that the OpenAPI type names were buggy. I think we're stuck with this until the next version.